src/server/server.py

```diff
@@ -314,7 +314,7 @@ async def start_client(
             self.log.debug(f"Connecting: {url}, Peer info: {target_node}")
             try:
                 ws = await session.ws_connect(
-                    url, autoclose=False, autoping=True, heartbeat=300, ssl=ssl_context, max_msg_size=50 * 1024 * 1024
+                    url, autoclose=True, autoping=True, heartbeat=60, ssl=ssl_context, max_msg_size=50 * 1024 * 1024
```

Comment:
Is @wjblanke correct that this may be too short for a harvester first load of 15000 plots?

Reply:
yes it's too short, however I have a feeling this is being done in another thread, because haitch was able to load 15k plots with no problem, which took over 10 minutes.